### ISSUE TITLE
Twig: Implement Local Navigation

### ISSUE THREAD

**Issue by justintemps** [international-labour-organization/designsystem#823]
## ⚠️ This can't be started until #1302 has also been completed. Once it's finished, both can be merged to production

This ticket will implement in Twig the refactoring of the Navigation component and introduction of the separate Local Navigation component carried out in #1302. 

Note that #1302 may not include styles for the `complex` version of the Local Navigation component. If that's the case, then this ticket should add the complex version to both Twig markdown and styles, and then create a new ticket to add the complex version to React.

[See designs for further reference](https://www.figma.com/design/RRxsfLNxjoypKrfJg1EwJf/ILO-Components-Library?m=auto&node-id=3979-4509&t=JIzo17dLg17Px4ez-1)

**Comment by bashlk** [comment]
@justintemps Do you have more specific suggestions for how this can be refactored? Perhaps split into two, similar to the navigation and local nav components?

**Comment by justintemps** [comment]
@bashlk exactly, it should be refactored so there are two different scss files, one for each component.

**Comment by justintemps** [comment]
@bashlk, @GGKapanadze I'm thinking this can wait for after the version 1. What do you think?

**Comment by bashlk** [comment]
@justintemps I don't think we have worked a lot with the Navigation component so far so it is hard for us to weigh in on this. But since this is an internal detail of the design system, I think it can be postponed.

**Comment by bashlk** [comment]
@justintemps We had a chat about this and we think that the Navigation component should probably be down in several separate components (e.g. language selector, search box, dropdown menu, horizontal nav). 

We also think that we should avoid having compound elements like the Navigation since they are not reusable and instead have simpler components like the ones listed above so that a consumer can combine them on their own.

**Comment by justintemps** [comment]
Hey @ademarco and @GGKapanadze, just to make sure you're aware I'm working on this.
